plugin/src/ios/CDVMParticle.m

```diff
@@ -69,6 +69,57 @@ - (void)setATTStatus:(CDVInvokedUrlCommand*)command {
     }];
 }
 
+- (void)addGDPRConsentState:(CDVInvokedUrlCommand*)command {
+    [self.commandDelegate runInBackground:^{
+        NSString *serializedConsent = [command.arguments objectAtIndex:0];
```

Comment:
Seems like use of 'id' may be hiding a bug.. this is declared as a string here but then gets treated as a dictionary inside the serialization method. Not sure if the type here needs to change or there's a missing call to deserialize json.